src/components/forms/AddressInput/index.tsx

```diff
@@ -38,14 +52,15 @@ const AddressInput = ({
     />
     <OnChange name={name}>
       {async (value) => {
-        if (isValidEnsName(value)) {
+        const address = trimSpaces(value)
+        if (isValidEnsName(address)) {
           try {
-            const resolverAddr = await getAddressFromENS(value)
+            const resolverAddr = await getAddressFromENS(address)
             fieldMutator(resolverAddr)
           } catch (err) {
             console.error('Failed to resolve address for ENS name: ', err)
           }
-        }
+        } else fieldMutator(address)
```

Comment:
what is this `else` for?

Reply:
if would resolve if address if an ENS is provided and set this value to the input.
else will set the value to the input, in this case after passing it through a trim function